types/rlp/decode.go

```diff
@@ -685,6 +696,21 @@ func decodeBigInt(s *Stream, val reflect.Value) error {
 	return nil
 }
 
+func decodeUint256(s *Stream, val reflect.Value) error {
+	// Set the integer bytes.
+	i := val.Interface().(*uint256.Int)
```

Comment:
I believe this panics, if its not of the type `*uint256.Int`.
You should catch this panic like this
```suggestion
	if ok, i := val.Interface().(*uint256.Int); ok {....}
```

Reply:
done